api/v1beta1/namespacedverifier_types.go

```diff
@@ -29,9 +29,13 @@ type NamespacedVerifierSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
```

Comment:
curious if you've run `make manifests generate` after this change?

Reply:
![image](https://github.com/user-attachments/assets/35765ef7-722a-4d16-8aa1-c06feb796f10)